features/git-prune-branches/main_branch/stale_local_feature_branch/with_open_changes.feature

```diff
@@ -22,3 +22,16 @@ Feature: git prune-branches: remove stale local feature branches when run on the
       | local      | main     |
       | remote     | main     |
       | coworker   | main     |
```

Comment:
This line can also be removed. Same change you did in the without_open_changes feature